src/components/Homepage/LendByCategory/KivaCardPromo.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="row kiva-card-promo">
+		<div class="small-12 xlarge-5 columns">
+			<!-- eslint-disable max-len -->
+			<svg class="kiva-card-promo__img" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 237 336">
```

Comment:
prob on the verge of being it's own file...but ok for now

Reply:
Yeah, I think I agree. It's halfway down the page, a lazy-loaded img tag would probably save some bytes